src/submission/submission-api.ts

```diff
@@ -259,6 +259,26 @@ class SubmissionController {
     const donorsJson = await fsPromises.readFile(req.file.path, 'utf-8');
     return res.status(201).send(await submission.operations.adminAddDonors(JSON.parse(donorsJson)));
   }
+
+  // @HasFullWriteAccess()
```

Comment:
missed this ?

Reply:
:eyes: will fix when I move this to clinical-api